crates/aptos-faucet/src/lib.rs

```diff
@@ -195,21 +195,23 @@ pub fn routes(
     health
         .or(mint)
         .with(warp::log::custom(|info| {
-            let actual_ip = info
+            let forwarded_for = info
                 .request_headers()
                 .get("x-forwarded-for")
                 .map(|inner| inner.to_str().unwrap_or("-"));
+
             info!(
-                "{} \"{} {} {:?}\" {} \"{}\" \"{}\" \"{}\" {:?}",
-                OptFmt(info.remote_addr()),
-                info.method(),
-                info.path(),
-                info.version(),
-                info.status().as_u16(),
-                OptFmt(info.referer()),
-                OptFmt(info.user_agent()),
-                OptFmt(actual_ip),
-                info.elapsed(),
+                remote_addr = info.remote_addr(),
+                forwarded_for = forwarded_for,
+                host = info.host(),
+                method = format!("{}", info.method()),
+                path = info.path(),
+                version = format!("{:?}", info.version()),
+                status = format!("{}", info.status()),
+                referer = info.referer(),
+                user_agent = info.user_agent(),
+                elapsed = info.elapsed(),
+                "mint request"
```

Comment:
yuck, I really hate this type of logging, but I guess we only have this service anyway...

Reply:
why though? can we at least agree that this is million times better than 10 random strings concatenated by spaces like it is right now? 😉

Reply:
Haha, I did approve, didn't I?

Reply:
It'd be good to get aligned with @banool on the logging since he will also have something for Tap.

Reply:
I have an issue in tap to do structured logging too, so this works for me.